packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -501,9 +506,14 @@ const ContextMenuSubContent = React.forwardRef<
       ref={forwardedRef}
       style={{
         ...props.style,
-        // re-namespace exposed content custom property
-        ['--radix-context-menu-content-transform-origin' as any]:
-          'var(--radix-popper-transform-origin)',
+        // re-namespace exposed content custom properties
+        ...{
+          '--radix-context-menu-content-transform-origin': 'var(--radix-popper-transform-origin)',
+          '--radix-context-menu-content-available-width': 'var(--radix-popper-available-width)',
+          '--radix-context-menu-content-available-height': 'var(--radix-popper-available-height)',
+          '--radix-context-menu-trigger-width': 'var(--radix-popper-anchor-width)',
```

Comment:
Similarly to the point above, should this be trigger or sub-trigger? Again I can see an argument for both.